extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ArcConfig.java

```diff
@@ -181,6 +181,13 @@
     @ConfigItem
     public ArcTestConfig test;
 
+    /**
+     * The list of packages that will not be checked for split package issues.
+     * Each element of the list has to be a String representation of full package name.
+     */
+    @ConfigItem
+    public Optional<List<String>> skipSplitPackageDetection;
```

Comment:
`skip-split-package-detection` reads as if it would disable it altogether, but I currently don't mave a much better suggestion:
`skip-split-package-detection-for` would make it even longer. `split-package-detection-exclusions` maybe?

Reply:
> skip-split-package-detection reads as if it would disable detection altogether

True, but it takes a list of strings so I hoped that would make it clearer. But I agree it is not perfect and all options are just too long :)

Reply:
How about `ignoredSplitPackages`, i.e. `quarkus.arc.ignored-split-packages`?

Reply:
Ah, very good catch! I always miss these...

Reply:
Updated PR with this suggestion